bin/peggy.mjs

```diff
@@ -360,6 +377,21 @@ readStream(inputStream, input => {
     });
   }
 
+  if (options.sourceMap) {
+    source = source.toStringWithSourceMap({
+      file: outputFile === "-" ? null : outputFile,
+    });
+
+    const sourceMapStream = fs.createWriteStream(progOptions.sourceMap);
+    sourceMapStream.on("error", () => {
+      abort(`Can't write to file "${progOptions.sourceMap}".`);
```

Comment:
I think you can get to this with the output going to a directory that doesn't exist.

Reply:
I don't think we should add missing directories, at least for now.